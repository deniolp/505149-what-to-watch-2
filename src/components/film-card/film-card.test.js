import React from 'react';
import renderer from 'react-test-renderer';

import FilmCard from './film-card';

describe(`FilmCard`, () => {
  const FilmCardMock = {
    name: `Aviator`,
    src: `img/aviator.jpg`,
  };

  it(`renders correctly`, () => {
    const tree = renderer.create(
        <FilmCard
          film={FilmCardMock}
          onCardMouseEnter={jest.fn()}
        />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
