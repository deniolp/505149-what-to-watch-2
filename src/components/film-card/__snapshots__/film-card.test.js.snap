// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmCard renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <a
    href="/film/1"
    onClick={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <video
        height="100%"
        muted={true}
        poster="/img/aviator.jpg"
        width="100%"
      >
        <source
          src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
          type="video/webm"
        />
      </video>
    </div>
  </a>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/film/1"
      onClick={[Function]}
    >
      Aviator
    </a>
  </h3>
</article>
`;
